src/test/java/mousio/etcd4j/TestFunctionality.java

```diff
@@ -113,6 +113,28 @@ public void testStoreStats() {
     assertNotNull(stats);
   }
 
+  /**
+   * Test Members
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testMembers() {
+    EtcdMembersResponse members = etcd.getMembers();
+    assertNotNull(members);
```

Comment:
If possible, it would be nice to test also the expected result of a call instead of just that the response is not null